cmake/test_util.cmake

```diff
@@ -50,17 +46,12 @@ function(create_standard_cxx_test)
   set(multiValues SOURCES LIBRARIES)
 
   include(CMakeParseArguments)
-  cmake_parse_arguments(${prefix} "${optionalValues}" "${singleValues}"
-                        "${multiValues}" ${ARGN})
+  cmake_parse_arguments(${prefix} "${optionalValues}" "${singleValues}" "${multiValues}" ${ARGN})
 
   add_executable(test_${TEST_NAME} ${TEST_SOURCES})
-  target_link_libraries(test_${TEST_NAME} PUBLIC 
-    LAPACK::LAPACK 
-    ${LAPACKE_LIBRARIES}
-    musica::tuvx
-    GTest::gtest_main
-  )
```

Comment:
I think this section is needed to link LAPACK correctly.

Reply:
Yes, while reverting the formatting changes I accidentally reverted this link. I have fixed this in my latest commit.